src/chocolatey.resources/helpers/chocolateyProfile.psm1

```diff
@@ -20,10 +20,9 @@ if (Get-Module chocolateyProfile) {
 
 $thisDirectory = (Split-Path -Parent $MyInvocation.MyCommand.Definition)
 
+Import-Module "$thisDirectory\Chocolatey.PowerShell.dll"

```

Comment:
Can we restrict which cmdlets are imported as part of the module?

We probably do not want all cmdlets to be available when `chocolateyProfile.psm1` is loaded.

Reply:
Hmm. _apparently_ we can 🤔 Will do.